setup.cfg

```diff
@@ -59,6 +59,8 @@ docs =
     # https://github.com/sphinx-doc/sphinx/issues/8198
     pygments >= 2.4.1
     linkify-it-py
+    phill @ git+https://github.com/snek5000/snek5000-phill.git
+    snek5000-cbox @ git+https://github.com/snek5000/snek5000-cbox.git@stable
```

Comment:
I recall that packaging for pypi does not permit this. So we have to either:

1. Push phill and cbox release
1. Use docs/requirements.txt file

Reply:
I removed phill and snek5000-cbox from setup.cfg. We would have to upload them to PyPI.

But then, this brings the question of the name of the package phill. I think snek5000-phill would be better.

Reply:
I agree. Let's change the name of the package. But perhaps retain the name of the top level module as `phill`. I cant imagine myself typing a long `import snek5000_phill` all the time and I dont want to give that impression that is the only way.